src/static/css/pages/careers.less

```diff
@@ -13,7 +13,19 @@
 
 .careers-working-at-cfpb {
     .content_hero {
-        min-height: 300px;
+        background-color: @white;
+        /* TODO: This breakpoint is set to the small image width,
+                 so that a gap does not open below the image. */
+        .respond-to-min(660px, {
+            min-height: 300px;
+        });
+
```

Comment:
Is there a better way to target the elements other than first-child/last-child? Maybe using the modifiers.